spec/rollout_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "spec_helper"
+require 'pry'
```

Comment:
Make sure to remove pry references before merging

Reply:
thanks, I added to work just locally